### PR TITLE
Integration 4.12.0 - Remerge Fixes

### PR DESCRIPTION
**Pull request by SaMuellerDLR**
Remerge of any fixes performed for the 4.12.0 Release.

### CODE REVIEW

File: de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java [hunk 1]
```diff
@@ -49,8 +46,8 @@
 	protected static final int MAX_TEST_CASE_TIMEOUT_SECONDS = 30;
 	protected static final int MAX_TEST_CASE_WAIT_TIME_MILLI_SECONDS = 1000;
 	
-	@Rule
-	public TestRule globalTimeout = new DisableOnDebug(Timeout.seconds(MAX_TEST_CASE_TIMEOUT_SECONDS));
+	//@Rule
+	//public TestRule globalTimeout = new DisableOnDebug(Timeout.seconds(MAX_TEST_CASE_TIMEOUT_SECONDS));
```
Comment (franzTobiasDLR) [inline de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java:50]: Do we want to disable this rule or was this just somehow for the release?

Comment (franzTobiasDLR) [reply de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java:50]: And if we don't want to have it why not deleting it?

Comment (SaMuellerDLR) [reply de.dlr.sc.virsat.project.test/src/de/dlr/sc/virsat/project/test/AProjectTestCase.java]: Nice catch. We only disabled it temporarily for debugging, its supposd to be renabled,

File: de.dlr.sc.virsat.build/src/de/dlr/sc/virsat/build/inheritance/AVirSatTransactionalBuilder.java [hunk 2]
```diff
@@ -163,6 +163,18 @@ protected void transactionalFullBuildRemoveProblemMarkers() {
 	protected void transactionalFullBuildUpdateProblemMarkers() {
 	}
 	
+	protected boolean saveAfterIncrementalBuild;
```
[the remark this answers is not in the document]
Comment (SaMuellerDLR) [reply de.dlr.sc.virsat.build/src/de/dlr/sc/virsat/build/inheritance/AVirSatTransactionalBuilder.java]: Made API clearer, inherting builders should only be able to set it to true so changed the name of the "setter" and made the variable private.

**Pull request merged**